lib/babe/crypto.go

```diff
@@ -58,25 +62,28 @@ func claimPrimarySlot(randomness Randomness,
 func checkPrimaryThreshold(randomness Randomness,
 	slot, epoch uint64,
 	output [sr25519.VRFOutputLength]byte,
-	threshold *common.Uint128,
+	threshold *scale.Uint128,
 	pub *sr25519.PublicKey,
-) bool {
+) (bool, error) {
 	t := makeTranscript(randomness, slot, epoch)
 	inout := sr25519.AttachInput(output, pub, t)
 	res := sr25519.MakeBytes(inout, 16, babeVRFPrefix)
 
-	inoutUint := common.Uint128FromLEBytes(res)
+	inoutUint, err := scale.NewUint128(res)
+	if err != nil {
+		return false, fmt.Errorf("failed to convert bytes to Uint128, %w", err)
```

Comment:
nit I know it's a super detail, but should we agree on a common way to wrap errors? I tend to use `context: %w` which seems like the common standard out there, but I'm happy to use any wrapping, as long as we make it the same in this codebase.

Reply:
will change it :+1:

Reply:
done